loadkeys -b's purpose is to dump a binary keymap
for usage with busybox on stdout, however it
stupidly prints other stuff as well, rendering
the binary kmap worthless. it can be fixed by
adding -q, however this should be implicitly
enabled by -b.


--- kbd-1.15.3.org/src/loadkeys.y
+++ kbd-1.15.3/src/loadkeys.y
@@ -1086,9 +1070,6 @@
 		case 'a':
 			opta = 1;
 			break;
-		case 'b':
-			optb = 1;
-			break;
 		case 'c':
 			nocompose = 1;
 			break;
@@ -1107,6 +1088,9 @@
 		case 'u':
 			optu = 1;
 			break;
+		case 'b':
+			optb = 1;
+			/* FALLTHRU */
 		case 'q':
 			quiet = 1;
 			break;
